### DISCUSSION TITLE
The ability to select a "core audio track" - Editing with multiple audio tracks

### DISCUSSION THREAD

**Discussion by Loafffy** [WyattBlue/auto-editor#469] [1. Help] [answered]
Auto editing footage with multiple audio tracks -It would be great if it were possible to select a single, or multiple audio channels to act as "core tracks". 

- - -

**For example:** I have a video of me playing a game. There are three audio tracks, _one microphone track, in game music, and a call with friends._

I select the _microphone track, and the call track_ as "core tracks" -the editor cuts out silences based purely on those tracks, ignoring the music track as reference entirely.

The result, _ideally_ is a video cut with only the gaps between conversations removed. No 'random' bits left in from louder parts in the music, or game sounds (which is the current result).

I'm not a programmer, but something like: `--use-tracks 1, 2`(?)

**Comment by WyattBlue** [comment, marked as answer]
In the readme:
```
# Different tracks can be set with different attribute.
auto-editor multi-track.mov --edit "(or audio:stream=0 audio:threshold=10%,stream=1)"
```

Only use the 2nd track:
```
auto-editor multi.mp4 --edit audio:stream=1
```
